docs/deep-dives/mobile/nimbus-cli/configuring-via-environment-var.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: nimbus-cli-environment-vars
+title: Configuration via environment variables
+slug: /nimbus-cli/environment-variables
+---
+
+## `adb` and `xcrun`
+
+The app supposes that `adb` and `xcrun` are on the `PATH`. If they are not (or aren't spelled `adb` or `adb.exe`), then you can configure them via the `ADB_PATH` and `XCRUN_PATH` environment variables.
+
+```sh
+ADB_PATH=C:\Program-Tools\bin\adb.exe nimbus-cli
```

Comment:
is this supposed to be `export ABD_PATH...`? Also, what is the `nimbus-cli` at the end doing? I don't think I understand this command